api/src/main/java/org/ehcache/CacheManager.java

```diff
@@ -25,10 +25,38 @@
  */
 public interface CacheManager {
 
+  /**
+   * Creates a {@link Cache} in this {@code CacheManager} according to the specified {@link CacheConfiguration}.
```

Comment:
I don't have a lot of javadoc experience really. Is there an argument for/against fully qualifying class names in @link refs?